gunicorn/pidfile.py

```diff
@@ -66,7 +66,10 @@ def validate(self):
             return
         try:
             with open(self.fname, "r") as f:
-                wpid = int(f.read() or 0)
+                try:
+                    wpid = int(f.read())
+                except ValueError:
+                    wpid = 0
```

Comment:
also the case where the pid is an empty file should be handled. I would do the following:
```
try:
    wpid = int(f.read() or 0)
except ValueError:
    raise HaltError("pidfile is invalid")
```

or any other error message. Thoughts?